### PR TITLE
Included  VLAN/VNI in available-resources network response

### PR DESCRIPTION
**Pull request by chaitra-mylarappachar**

### CODE REVIEW

**Review by dbozzato81 — APPROVED** [review]
LGTM

File: v1/pkg/client/api/openapi.yaml [hunk 1]
```diff
@@ -2695,19 +2695,23 @@ components:
     AvailableResources:
       example:
         Networks:
-        - IPPoolID: IPPoolID
+        - VNI: 6
```
Comment (mchuang3) [inline v1/pkg/client/api/openapi.yaml:2698]: Not a big deal, but VNIs are typically large numbers. Is there a way to specify example values? VNI: 15006 would be more common.

File: v1/pkg/client/api/openapi.yaml [hunk 2]
```diff
@@ -2695,19 +2695,23 @@ components:
...
+          IPPoolID: IPPoolID
           Description: Private network for intra-cluster communication
           LocationID: 046b6c7f-0b8a-43b9-b35d-6489e6daee91
           HostUse: Default
           Purpose: Storage
           ID: 046b6c7f-0b8a-43b9-b35d-6489e6daee91
+          VLAN: 0
```
Comment (mchuang3) [inline v1/pkg/client/api/openapi.yaml:2705]: This is definitely not right. VNI may be 0 for non-EVPN network types, but VLAN ID will always be non-zero. Would be better to show this VLAN with example values like 200.

**Review by mchuang3 — COMMENTED** [review]
Sorry, a little late to this review. I think client files are generated, but would be good to get correct example values.  Thanks.

**Pull request merged**